yaccparser.py

```diff
@@ -711,6 +713,47 @@ def p_multiplyinstruction(p):
     # so we just pack it in a bytes object
     p[0] = (struct.pack("<I", p[0]), None)
 
+def p_multiplylonginstruction(p):
+    """multiplylonginstruction : OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA REG COMMA REG
+                               | OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA SHARP CONST"""
+    global currentMnemonic
+    if len(p) == 10:
+        raise YaccError("Une instruction {} ne peut recevoir de constante comme dernier argument, seulement un registre.".format(currentMnemonic))
+
+    p[0] = 9 << 4
+    p[0] |= 1 << 23
+    assert len(p) == 11         # Check if we have 4 registers
+
+    regs = [p[4], p[6], p[8], p[10]]
+    # R15 (PC) must not be used as an operand or as a destination register
+    if 15 in regs:
+            raise YaccError("Le régistre PC ne peut pas être utilisé.")
```

Comment:
This indentation is 8 spaces instead of 4 like the rest of the file. This should be fixed. Python will understand, but keeping the file coherent is important for readability, I believe.